library/common/types/utility.cc

```diff
@@ -0,0 +1,27 @@
+#include <string>
+
+#include "library/common/data/utility.h"
+
+namespace Envoy {
+namespace Types {
+
+envoy_map makeEnvoyMap(std::vector<std::pair<std::string, std::string>> pairs) {
```

Comment:
this method probably belong here: https://github.com/envoyproxy/envoy-mobile/blob/main/library/common/data/utility.h